aspnetcore/performance/caching/memory/sample/WebCache/Controllers/HomeController.cs

```diff
@@ -46,6 +46,33 @@ public IActionResult CacheTryGetValueSet()
     }
     #endregion
 
+    #region snippetSize
+    public IActionResult CacheTryGetValueSetSize()
+    {
+        DateTime cacheEntry;
+
+        // Look for cache key.
+        if (!_cache.TryGetValue(CacheKeys.Entry, out cacheEntry))
+        {
+            // Key not in cache, so get data.
+            cacheEntry = DateTime.Now;
+
+            // Set cache options.
+            var cacheEntryOptions = new MemoryCacheEntryOptions()
+                .SetSize(1024*2)
+                // Keep in cache for this time, reset time if accessed.
+                .SetSlidingExpiration(TimeSpan.FromSeconds(3));
+
+            cacheEntryOptions.Size = 1;
```

Comment:
Comment one of these out, don't set it twice.
Also, setting it to different values is confusing.